pkg/docker/checks.go

```diff
@@ -1,7 +1,7 @@
 package docker
 
 const (
-	ACRegistry   = "appscode"
+	ACRegistry   = "emruzhossain"
```

Comment:
You can set it [here](https://github.com/appscode/stash/blob/master/test/e2e/e2e_suite_test.go#L72) for running e2e tests.